caddytls/certificates.go

```diff
@@ -99,7 +100,20 @@ func (cfg *Config) CacheManagedCertificate(domain string) (Certificate, error) {
 	if err != nil {
 		return Certificate{}, err
 	}
-	cert, err := makeCertificate(siteData.Cert, siteData.Key)
+
+	ctLogURLs := cfg.CTLogURLs
+	if len(ctLogURLs) == 0 {
+		// TODO: Venafi was just distrusted; one idea would be to fetch the
+		// Chrome list of trusted logs and just submit to everything until we
+		// have 1 Google and 1 non-Google. This would be hilarious, and pretty
+		// resillient.
+		ctLogURLs = []string{
```

Comment:
> [...] Venafi was just distrusted [...]

If Venafi is distrusted we should replace it with another log!

> one idea would be to fetch the Chrome list of trusted logs and just submit to everything until we have 1 Google and 1 non-Google. This would be hilarious, and pretty

This sounds amazing 😄